### ISSUE TITLE
Curriculum Face

### ISSUE THREAD

**Issue by octavianChen** [deepinsight/insightface#1947]
May be the loss.py only cover loss functions which revise the positive example's logits(such as arcface and cosface). If I want revise the postive examle's logits and negative example's logits(such as curriculum face), how? Thank you

**Comment by wzhiyuan2016** [comment]
以前想到过这个问题，刚看到，可使用distributed.all_gather，搜集所有的GPU卡上的 logis即可，然后再算target_logit值。
